src/components/queryBuilder/FilterEditor.tsx

```diff
@@ -329,38 +332,44 @@ export const FilterEditor = (props: {
     onFilterChange(index, filter);
   };
   return (
-    <EditorFieldGroup>
+    <>
       {index !== 0 && (
         <RadioButtonGroup options={conditions} value={filter.condition} onChange={(e) => onFilterConditionChange(e!)} />
       )}
       <Select
+        disabled={Boolean(filter.hint)}
```

Comment:
Would it be more clear to have a specific prop/state to handle disabling the Select?

Reply:
I suppose so, but this component is only used within this file, so it would be the same line moved elsewhere. Might make the tests clearer though